File: frontend/src/app/api-generated/models/user-booking-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { BookingPlaceViewModel } from '../models/booking-place-view-model';
import { BookingStateEnum } from '../models/booking-state-enum';
export interface UserBookingViewModel {
  bookedById?: number | null;
  bookingDate?: string;
  bookingId?: number;
  bookingPlaceId?: number;
  bookingPlaceVm?: BookingPlaceViewModel;
  state?: BookingStateEnum;
}

File: backend/ViewModels/UserBookingViewModel.cs
﻿using BookingApp.Enums;

namespace BookingApp.ViewModels
{
    public class UserBookingViewModel
    {
        public int BookingId { get; set; }
        public int BookingPlaceId { get; set; }
        public BookingStateEnum State { get; set; }
        public DateTime BookingDate { get; set; }
        public int? BookedById { get; set; }
        public BookingPlaceWithBookingsViewModel? BookingPlaceVm { get; set; }
    }
}
